app/(dashboard)/users/page.tsx

```diff
@@ -20,7 +20,17 @@ export default async function UsersPage() {
         icon={mdiAccountSupervisor}
         createHref='./register'
       />
-      <UsersList page={page} />
+      <List
+        headers={{
+          ID:       'id',
+          Nombre:   'firstName',
+          Apellido: 'lastName',
+          Email:    'email',
+          Rol:      'role',
+        }}
```

Comment:
Can we define a separate object instead of hardcoding these values?

Reply:
Dale